qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -257,6 +252,23 @@ def _draw_widgets(self):
                 self.prog.enabled = True
                 if tab:
                     self.prog.on_tab_changed(tab)
+            elif segment.startswith('text'):
```

Comment:
```suggestion
            elif segment.startswith('text:'):
```

(doesn't make a difference since we validate the values anyways, but just seems a bit clearer to me)